openpype/hosts/max/plugins/publish/extract_tycache.py

```diff
@@ -0,0 +1,197 @@
+import os
+
+import pyblish.api
+from pymxs import runtime as rt
+
+from openpype.hosts.max.api import maintained_selection
+from openpype.pipeline import publish
+
+
+class ExtractTyCache(publish.Extractor):
+    """
+    Extract tycache format with tyFlow operators.
+    Notes:
+        - TyCache only works for TyFlow Pro Plugin.
+
+    Args:
+        self.export_particle(): sets up all job arguments for attributes
+            to be exported in MAXscript
+
+        self.get_operators(): get the export_particle operator
+
+        self.get_files(): get the files with tyFlow naming convention
+            before publishing
+    """
+
+    order = pyblish.api.ExtractorOrder - 0.2
+    label = "Extract TyCache"
+    hosts = ["max"]
+    families = ["tycache"]
+
+    def process(self, instance):
+        # TODO: let user decide the param
+        start = int(instance.context.data.get("frameStart"))
```

Comment:
Unsure if "frameStart" would be ever `None` in which instance it would fail to `int()`

Reply:
With what @Minkiu says here the case of `frameStart` not being defined would error - so we're just as well of assuming it exists:
```suggestion
        start = instance.context.data["frameStart"])
```

Aside of that - why are we using the **context** start frame and not a start frame of the instance? 😮